class_project_instructions/README.md

```diff
@@ -190,9 +190,9 @@ Before starting implementation, you need to choose **one** of the two supported
 Docker-based workflows. Finalize your setup choice before proceeding with
 development.
 
-### 1. Simple Docker Setup (`data605_style`) — Recommended for Students
+### 1. Simple Docker Setup (`coursecode_style`) — Recommended for Students
```

Comment:
Maybe we should call the 2 styles: "simple" vs "causify_dev_system" so it's clearer